website/docs/r/network_peering.html.markdown

```diff
@@ -69,65 +75,99 @@ resource "aws_vpc_peering_connection_accepter" "peer" {
-  peer_network = "projects/${mongodbatlas_network_peering.test.atlas_gcp_project_id}/global/networks/${mongodbatlas_network_peering.test.atlas_vpc_name}"
+resource "mongodbatlas_cluster" "test" {
+  project_id   = "${local.project_id}"
+  name         = "terraform-manually-test"
+  num_shards   = 1
+  disk_size_gb = 5
+
+  replication_factor           = 3
+  auto_scaling_disk_gb_enabled = true
+  mongo_db_major_version       = "4.0"
+
+  //Provider Settings "block"
+  provider_name               = "GCP"
+  provider_instance_size_name = "M10"
+  provider_region_name        = "US_EAST_4"
+
+  depends_on = ["google_compute_network_peering.peering"]
 }
+
+#  Private connection strings are not available w/ GCP until the reciprocal
+#  connection changes to available (i.e. when the status attribute changes
+#  to AVAILABLE on the 'mongodbatlas_network_peering' resource, which
+#  happens when the google_compute_network_peering and and
+#  mongodbatlas_network_peering make a reciprocal connection).  Hence
+#  since the cluster can be created before this connection completes
+#  you may need to run `terraform refresh` to obtain the private connection strings.  
+
 ```
 
 ### Example with Azure
 
 ```hcl
 
 resource "mongodbatlas_network_container" "test" {
-  project_id       = local.project_id
-  atlas_cidr_block = "10.8.0.0/21"
+  project_id       = "${local.project_id}"
+  atlas_cidr_block = "192.168.208.0/21"
   provider_name    = "AZURE"
-  region           = "US_WEST"
-}
-
-resource "mongodbatlas_private_ip_mode" "my_private_ip_mode" {
-  project_id = "${mongodbatlas_project.my_project.id}"
-  enabled    = true
+  region           = "US_EAST_2"
 }
 
 resource "mongodbatlas_network_peering" "test" {
-  project_id            = local.project_id
-  atlas_cidr_block      = "10.8.0.0/21"
-  container_id          = mongodbatlas_network_container.test.container_id
+  project_id            = "${local.project_id}"
+  atlas_cidr_block      = "192.168.0.0/21"
+  container_id          = "${mongodbatlas_network_container.test.container_id}"
   provider_name         = "AZURE"
-  azure_directory_id    = "35039750-6ebd-4ad5-bcfe-cb4e5fc2d915"
-  azure_subscription_id = "g893dec2-d92e-478d-bc50-cf99d31bgeg9"
-  resource_group_name   = "atlas-azure-peering"
-  vnet_name             = "azure-peer"
+  azure_directory_id    = "${local.AZURE_DIRECTORY_ID}"
+  azure_subscription_id = "${local.AZURE_SUBCRIPTION_ID}"
+  resource_group_name   = "${local.AZURE_RESOURCES_GROUP_NAME}"
+  vnet_name             = "${local.AZURE_VNET_NAME}"
+}
+
+resource "mongodbatlas_cluster" "test" {
+  project_id = "${local.project_id}"
+  name       = "terraform-manually-test"
+  num_shards = 1
+
+  replication_factor           = 3
+  auto_scaling_disk_gb_enabled = true
+  mongo_db_major_version       = "4.0"
 
-  depends_on = [mongodbatlas_private_ip_mode.my_private_ip_mode]
+  //Provider Settings "block"
+  provider_name               = "AZURE"
+  provider_disk_type_name     = "P6"
```

Comment:
could probably start with a P4 for 32GB storage only

Reply:
makes sense, that's the default.  Will change